daemon/algod/api/algod.oas2.json

```diff
@@ -4329,10 +4373,14 @@
         "unnamed-resources-accessed": {
           "$ref": "#/definitions/SimulateUnnamedResourcesAccessed"
         },
-        "fixed-signer":{
+        "fixed-signer": {
           "description": "The account that needed to sign this transaction when no signature was provided and the provided signer was incorrect.",
           "type": "string",
           "x-algorand-format": "Address"
+        },
+        "populated-resource-arrays": {
```

Comment:
I'm basically agnostic on this question: Do you think this should be present if it's empty? I lean away toward to appearing, but don't care too much.

Reply:
I think mirroring the behavior of `unnamed-resources-accessed` (not being a required field) makes the most sense